packages/world/script/initObjects.sol

```diff
@@ -539,6 +539,15 @@ function initObjects() {
     ObjectTypes.WheatSeeds,
     ObjectTypeMetadataData({ stackable: 99, maxInventorySlots: 0, mass: 0, energy: 100, canPassThrough: true })
   );
+  // TODO: tree seeds must match the total energy of its components
```

Comment:
I assume we'll pick some energy value based on the average tree. Then we return the remaining to energy pool anyways

Reply:
Yeah exactly, as long as it is higher or equal to the maximum possible energy for the tree this will work

Reply:
with the randomness stuff we'll know the max possible tree size right?

Reply:
The maximum size won't depend on randomness, I can write a formula later given the tree params